recipes-support/librealsense/librealsense_1.10.0.bb

```diff
@@ -88,3 +88,10 @@ FILES_${PN}-tests += "\
 
 # Configuration is not required
 do_configure[noexec] = "1"
+
+#Skip ldflags QA check
```

Comment:
@mythi I tried by passing LDFLAGS to make as below(in bbappend):
    EXTRA_OEMAKE_append = " 'LDFLAGS=${LDFLAGS}'"
Still that couldn't solve the issue :(